parm/parm_gdas/aero_crtm_coeff.yaml

```diff
@@ -1,23 +1,23 @@
 mkdir:
 - !ENV ${DATA}/crtm/
 copy:
-- - !ENV ${FV3JEDI_FIX}/crtm/${CRTM_VER}/AerosolCoeff.bin
+- - !ENV ${FV3JEDI_FIX}/crtm/${crtm_VERSION}/AerosolCoeff.bin
   - !ENV ${DATA}/crtm/AerosolCoeff.bin
```

Comment:
```suggestion
- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/AerosolCoeff.bin $(DATA)/crtm]
- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/CloudCoeff.bin    $(DATA)/crtm]
```

Reply:
Here is a simple edit.
translate `${ }` to `$( )` and one does not need a destination file if the source file is the same name.

Reply:
The suggested change made but I'm not certain if it is correct.  My working copy of `aero_crtm_coeff.yaml` now reads

```
mkdir:
- !ENV ${DATA}/crtm/
copy:
- $(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/AerosolCoeff.bin
- $(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/CloudCoeff.bin
- $(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_npp.SpcCoeff.bin
- $(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_npp.TauCoeff.bin
- $(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_j1.SpcCoeff.bin
- $(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_j1.TauCoeff.bin
- $(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISice.EmisCoeff.bin
- $(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISland.EmisCoeff.bin
- $(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISsnow.EmisCoeff.bin
- $(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISwater.EmisCoeff.bin
```

Seems the leading `- !ENV ${DATA}/crtm/` should also go, but what does it become?   Does it become `- $(DATA)/crtm/`?

Reply:
Those 2 lines are combined to a single one.
```
- - !ENV ${FV3JEDI_FIX}/crtm/${crtm_VERSION}/AerosolCoeff.bin
  - !ENV ${DATA}/crtm/AerosolCoeff.bin
```
becomes
```
- $(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/AerosolCoeff.bin $(DATA)/crtm/
```

Reply:
Thanks, here's round 2
```
mkdir:
- $(DATA)/crtm/
copy:
- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/AerosolCoeff.bin. $(DATA)crtm/]
- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/CloudCoeff./bin, $(DATA)/crtm/]
- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_npp.SpcCoeff./bin, $(DATA)/crtm/]
- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_npp.TauCoeff./bin, $(DATA)/crtm/]
- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_j1.SpcCoeff./bin, $(DATA)/crtm/]
- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_j1.TauCoeff./bin, $(DATA)/crtm/]
- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISice.EmisCoeff./bin, $(DATA)/crtm/]
- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISland.EmisCoeff./bin, $(DATA)/crtm/]
- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISsnow.EmisCoeff./bin, $(DATA)/crtm/]
- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISwater.EmisCoeff./bin, $(DATA)/crtm/]
```

Reply:
the first line has a `.` instead of `,`

Reply:
All the lines with`./bin` are wrong.   It should be `.bin`.   This has been corrected.   The period on the first line was replaced with a comma.   Committed to `feature/refactor_jediaero` at a022aefb.